src/Components/HelpBubbleIcon/HelpButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { IconButton } from '@mui/material';
+import { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { HelpBubbleIcon } from './helpBubbleIcon.tsx';
+
+const HelpButton = ({
+  className,
+  helpText,
+  helpId,
+  isVisible,
+}: {
+  className?: string;
+  helpText: string;
+  helpId?: string;
```

Comment:
Why is the `helpId` optional?

Reply:
Was not sure if you always have it, changed it to required